### DISCUSSION TITLE
Issue with verification

### DISCUSSION THREAD

**Discussion by aentreserra** [community/community#139326] [GitHub Education]
### Hi everyone,

I'm trying to verify my student status, but I'm encountering some issues. The first problem is that when I select the facing back camera on my phone, the screen goes black, and I can't take pictures. The photos taken with the front camera don't focus properly, and the documents are not clear. Additionally, my documents are in Spanish, and as far as I can see, they need to be submitted in both the original language and in English, but I can only attach one image.

Furthermore, I’m studying online, as stated on my student ID card, but they aren’t verifying my status. Even though the ID clearly says "online," they are not recognizing it.

Thanks 🥺

**Comment by Y3su** [comment]
Hey, @aentreserra! Try these tips that might solve your problem.

1. Camera Problems:
   If your phone’s back camera keeps going black, try restarting it, or use another device (like a friend’s phone or a tablet) to take a picture. You can also try using a document scanning app (like CamScanner or Adobe Scan) with the front camera if that’s working better. They usually make things clearer and focus better on text!

2. Documents in Spanish:
   As for the documents, if they need both the original and the English translation, you can merge them into one file. You can use something like PDF Merge or even a photo editing app to put both images side by side or combine them into a single file.

3. Online Student Status:
   Since your ID mentions "online," but they’re not accepting it, you might want to add extra proof, like an official letter from your school that confirms your enrollment. Sometimes, having a bit more info helps them verify your status faster.

4. Contact GitHub:
   If you’re still stuck, don’t hesitate to reach out to GitHub support. They can walk you through it and might be able to sort things out directly.

**Reply by aentreserra** [reply]
Thanks, I will try
